src/bin/pg_autoctl/pgsql.c

```diff
@@ -2260,6 +2269,13 @@ parsePgReachedTargetLSN(void *ctx, PGresult *result)
 		return;
 	}
 
+	if (PQntuples(result) == 0)
+	{
+		log_warn("Query returned no rows");
```

Comment:
This vague `log_warn` seems a bit unnecessary, given we already have the clearer later in the code: "No standby nodes are connected at the moment"

Reply:
Good point, going to make it a debug or trace message.